templates/cicd/buildspec.yml

```diff
@@ -0,0 +1,74 @@
+# Buildspec run in the build stage of your pipeline.
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      docker: 18
+      golang: 1.13
+      nodejs: 10
+      ruby: 2.6
+    commands:
+      # Download the CLI.
+      - export GOMOD=$CODEBUILD_SRC_DIR/go.mod:GOMOD
+      - export GOPATH="/go"
+      - echo "cd into $CODEBUILD_SRC_DIR"
+      - cd $CODEBUILD_SRC_DIR
+      - git clone https://github.com/aws/amazon-ecs-cli-v2
```

Comment:
could you add a TODO to download the latest tagged commit?

Reply:
I don't know if the latest tagged is what we want to do yet, but added this todo:
`# TODO download a validated version of the CLI instead of latest from master.`